microsoft-azure-sb/3.0/modules/ROOT/pages/azure-service-bus-connector-reference.adoc

```diff
@@ -0,0 +1,508 @@
+| Time To Live Time Unit a| Enumeration, one of:
+
+** NANOSECONDS
+** MICROSECONDS
+** MILLISECONDS
+** SECONDS
+** MINUTES
+** HOURS
+** DAYS | Time unit for the Write Concern Timeout |  SECONDS | 
+| Scheduled Enqueue Time Utc a| DateTime |  Defines the UTC instant at which the message is logically enqueued, sequenced, and therefore made available for retrieval |  | 
+| Properties a| Object |  Map of user application properties set on the message |  | 
+| Reconnection Strategy a| * <<reconnect>>
+* <<reconnect-forever>> | Retry strategy in case of connectivity errors |  | 
+|===
+
+
+=== For Configurations
+* <<config>> 
+
+==== Throws
+* AZURE-SERVICE-BUS-MESSAGING:PAYLOAD_SIZE_EXCEEDED 
+* AZURE-SERVICE-BUS-MESSAGING:OPERATION_TIMEOUT 
+* AZURE-SERVICE-BUS-MESSAGING:INVALID_SIGNATURE 
+* AZURE-SERVICE-BUS-MESSAGING:NOT_ENOUGH_PERMITS 
+* AZURE-SERVICE-BUS-MESSAGING:CONNECTIVITY 
+* AZURE-SERVICE-BUS-MESSAGING:DESTINATION_NOT_FOUND 
+* AZURE-SERVICE-BUS-MESSAGING:RETRY_EXHAUSTED 
+* AZURE-SERVICE-BUS-MESSAGING:INVALID_PARAMS
+* MULE:ANY  
+
+
+[[sendMessageBatch]]
+=== Send batch of messages
+`<azure-service-bus-messaging:send-message-batch>`
+
+This operation sends a list of messages to a queue or topic.
+
+
+==== Parameters
+[%header,cols="20s,20a,35a,20a,5a"]
+|===
+| Name | Type | Description | Default Value | Required
+| Configuration | String | Name of the configuration to use | | x
+| Destination Name a| String | Destination name to which to send the message |  | x
+| Messages a| Array of Any |  Messages to send |  #[payload] | 
+| Reconnection Strategy a| * <<reconnect>>
+* <<reconnect-forever>> |  A retry strategy in case of connectivity errors |  | 
+|===
+
+
+=== For Configurations
+* <<config>> 
+
+==== Throws
+* AZURE-SERVICE-BUS-MESSAGING:PAYLOAD_SIZE_EXCEEDED 
+* AZURE-SERVICE-BUS-MESSAGING:OPERATION_TIMEOUT 
+* AZURE-SERVICE-BUS-MESSAGING:INVALID_SIGNATURE 
+* MULE:ANY 
+* AZURE-SERVICE-BUS-MESSAGING:NOT_ENOUGH_PERMITS 
+* AZURE-SERVICE-BUS-MESSAGING:CONNECTIVITY 
+* AZURE-SERVICE-BUS-MESSAGING:DESTINATION_NOT_FOUND 
+* AZURE-SERVICE-BUS-MESSAGING:RETRY_EXHAUSTED 
+* AZURE-SERVICE-BUS-MESSAGING:INVALID_PARAMS 
+
+
+== Sources
+
+[[message-listener]]
+=== Message Listener
+`<azure-service-bus-messaging:message-listener>`
+
+This operation receives messages from a specific queue or subscription.
+
+
+==== Parameters
+[%header,cols="20s,20a,35a,20a,5a"]
+|===
+| Name | Type | Description | Default Value | Required
+| Configuration | String | Name of the configuration to use. | | x
+| Acknowledgment mode a| Enumeration, one of:
+
+** AUTO
+** MANUAL
+** IMMEDIATE |  |  | x
+| Prefetch Count a| Number |  When Prefetch is enabled, the receiver quietly acquires more messages, up to the PrefetchCount limit, beyond what the application initially requested. Setting this value to zero (`0`) disables prefetching. |  0 | 
```

Comment:
For later - what does "quietly" mean in this context?

Reply:
It's the same as silently, which is common and widely known terminology in computing. But Azure documentation says "quietly" so I'm going to leave it that way to align with Azue terminology.